packages/fx-core/src/plugins/solution/fx-solution/arm.ts

```diff
@@ -67,6 +67,8 @@ const bicepOrchestrationConfigFileName = "config.bicep";
 const templatesFolder = "./templates/azure";
 const configsFolder = `.${ConfigFolderName}/configs`;
 const parameterFileNameTemplate = `azure.parameters.${EnvNamePlaceholder}.json`;
+const pollWaitSeconds = 10;
+const MaxLogTimes = 4;
```

Comment:
`m`axLogTimes to make the variable name aligned
by the way, can we have a better name for it? not sure whether `maxRetryTimes` suits for it.

Reply:
It is fixed. Thanks.